### PR TITLE
refactor(dms/rocketmq_consumerGroup): fix dms rocketmq consumer group lint issues

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:
  fix dms rocketmq consumer group lint issues
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
  fix dms rocketmq consumer group lint issues
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/dms/' TESTARGS='-run TestAccDmsRocketMQConsumerGroup_basic'        
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/dms/ -v -run TestAccDmsRocketMQConsumerGroup_basic -timeout 360m -parallel 4 
=== RUN   TestAccDmsRocketMQConsumerGroup_basic 
=== PAUSE TestAccDmsRocketMQConsumerGroup_basic 
=== CONT  TestAccDmsRocketMQConsumerGroup_basic
--- PASS: TestAccDmsRocketMQConsumerGroup_basic (717.06s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dms       717.107s
```

### CODE REVIEW

File: huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_consumer_group.go [hunk 1]
```diff
@@ -190,7 +195,7 @@ func resourceDmsRocketMQConsumerGroupUpdate(ctx context.Context, d *schema.Resou
 	return resourceDmsRocketMQConsumerGroupRead(ctx, d, meta)
 }
 
-func buildUpdateRocketmqConsumerGroupBodyParams(d *schema.ResourceData, config *config.Config) map[string]interface{} {
+func buildUpdateRocketmqConsumerGroupBodyParams(d *schema.ResourceData, _ *config.Config) map[string]interface{} {
```
Comment (Lance52259) [inline huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_consumer_group.go]: If the config is not used, we should remove it from the argument list.

File: huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_consumer_group_test.go [hunk 2]
```diff
@@ -156,11 +163,13 @@ func testDmsRocketMQConsumerGroup_basic_update(name string) string {
 %s
 
 resource "huaweicloud_dms_rocketmq_consumer_group" "test" {
-  instance_id    = huaweicloud_dms_rocketmq_instance.test.id
-  broadcast      = false
+  instance_id = huaweicloud_dms_rocketmq_instance.test.id
+  broadcast   = false
+
   brokers        = [
```
Comment (chengxiangdong) [inline huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_consumer_group_test.go]:
```suggestion
  brokers = [
```

File: huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_consumer_group.go (hunk 1 above)
Comment (houpeng80) [reply huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_consumer_group.go]: done

File: huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_consumer_group_test.go (hunk 2 above)
Comment (houpeng80) [reply huaweicloud/services/acceptance/dms/resource_huaweicloud_dms_rocketmq_consumer_group_test.go]: done

**Pull request merged**